atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/ColumnValueDescription.java

```diff
@@ -253,38 +247,6 @@ public String getPersistCode(String varName) {
                 + "com.palantir.atlasdb.table.description.ColumnValueDescription.Compression." + compression + ")";
     }
 
-    public byte[] persistJsonToBytes(String str) throws ParseException {
-        return persistJsonToBytes(Thread.currentThread().getContextClassLoader(), str);
-    }
-
-    @SuppressWarnings("unchecked")
-    public byte[] persistJsonToBytes(ClassLoader classLoader, String str) throws ParseException {
-        final byte[] bytes;
-        if (format == Format.PERSISTABLE) {
-            throw new SafeIllegalArgumentException("Tried to pass json into a persistable type.");
-        } else if (format == Format.PERSISTER) {
-            Persister<?> persister = getPersister();
-            if (JsonNode.class == persister.getPersistingClassType()) {
-                try {
-                    JsonNode jsonNode = new ObjectMapper().readValue(str, JsonNode.class);
-                    return ((Persister<JsonNode>) persister).persistToBytes(jsonNode);
-                } catch (IOException e) {
-                    throw Throwables.throwUncheckedException(e);
-                }
-            } else {
-                throw new SafeIllegalArgumentException("Tried to write json to a Persister that isn't for JsonNode.");
-            }
-        } else if (format == Format.PROTO) {
-            Message.Builder builder = createBuilder(classLoader);
-            // This will have issues with base64 blobs
-            JsonFormat.merge(str, builder);
```

Comment:
this method was changed in one of the dependency bumps, which is why my attention was drawn to this class.